joeynmt/prediction.py

```diff
@@ -268,19 +268,25 @@ def test(cfg_file,
     Main test function. Handles loading a model from checkpoint, generating
     translations and storing them and attention plots.
 
-    :param cfg_file: path to configuration file
+    :param cfg_file: path to configuration file. Config file can be the yaml file or the loaded config file
```

Comment:
This ambiguity would be helpful to express in the [typing information](https://docs.python.org/3/library/typing.html#) above in line 261.